tests/ui/async-await/async-closures/without-precise-captures-we-are-powerless.stderr

```diff
@@ -29,7 +29,7 @@ LL |     outlives::<'a>(call_once(c));
 LL | }
    | - `c` dropped here while still borrowed
    |
-note: requirement that `c` is borrowed for `'a` introduced here
+note: requirement for `'a` introduced here
```

Comment:
what is a requirement for `'a`. I feel like the requirement is *for the APIT* here, not `'a`. Is it not possible to keep sth like "requirement that <whatever> outlives <region>" introduced here"?